flax/nn/attention.py

```diff
@@ -19,17 +19,19 @@
 import warnings
 
 from .. import jax_utils
-from . import base
-from . import initializers
-from . import stochastic
+from . import activation  # pytype: disable=pyi-error
+from . import base  # pytype: disable=pyi-error
+from . import initializers  # pytype: disable=pyi-error
+from . import stochastic  # pytype: disable=pyi-error
+from .linear import default_kernel_init  # pytype: disable=pyi-error
+from .linear import DenseGeneral  # pytype: disable=pyi-error
```

Comment:
Perhaps doing it like this reads nicer? (Also in many other places.)

```suggestion
# pytype: disable=attribute-error
from . import activation
from . import base
from . import initializers
from . import stochastic
from .linear import default_kernel_init
from .linear import DenseGeneral
# pytype: enable=attribute-error
```

Reply:
I kind of like the line-by-line thing. The regions makes it easy to be lazy and add imports in the section that don't actually need the escape hatch